format/midi/constants.go

```diff
@@ -0,0 +1,369 @@
+package midi
+
+import (
+	"github.com/wader/fq/pkg/scalar"
+)
+
+var notes = scalar.UintMapSymStr{
+	uint64(127): "G9",
```

Comment:
I think you can skip the casts here?

Reply:
Huh! Weird - at some point the compiler was insisting on the cast. 

Fixed in [17bac77](https://github.com/wader/fq/pull/1004/commits/17bac771df762860471e0f2b088fcdbe167ba00b).